Local Pods/GitHubAPI/GitHubAPI/V3SubscribeThreadRequest.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  V3SubscribeThreadRequest.swift
+//  GitHubAPI
+//
+//  Created by Ryan Nystrom on 6/10/18.
+//  Copyright © 2018 Ryan Nystrom. All rights reserved.
+//
+
+import Foundation
+
+public struct V3SubscribeThreadRequest: V3Request {
+    public typealias ResponseType = V3StatusCodeResponse<V3StatusCode200>
+    public var pathComponents: [String] {
+        return ["notifications", "threads", id, "subscription"]
+    }
+    public var method: HTTPMethod { return .put }
+    public var parameters: [String : Any]? {
+        return [
```

Comment:
Why not doing the same as for the `pathComponents` ?

Reply:
What do you mean?

Reply:
Doing it like this
```swift 
return ["ignored": ignore ? "true" : "false"]
```